collections/testtscripts.js

```diff
@@ -68,6 +68,9 @@ Meteor.methods({
 
     if (failers[0] != undefined && failers.length > 0) {
       status = 'fail';
+      if (Meteor.isServer) {
```

Comment:
as per our convo, this call should live where the actual event is being called. So, you will need to create a new meteor method.